tests/test_orphan_child_cleanup.py

```
@@ -0,0 +1,106 @@
+
+import pytest
+
+from mnemosyne.core.beam import BeamMemory
+
+
+@pytest.fixture
+def temp_db(tmp_path: Path) -> Path:
+    return tmp_path / "validate_delete_test.db"
+
+
+def _seed_embedding(conn, memory_id: str, vector: str = "[0.1, 0.2]"):
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `_seed_embedding`

Add return type annotation: `None`

(ANN202)